frontend/src/components/common/Stepper.tsx

```diff
@@ -2,12 +2,17 @@ import React, { Fragment } from 'react';
 import classNames from 'classnames';
 import { FaCheck } from 'react-icons/fa';
 import './Stepper.scss';
+import { ObjectSchema } from 'yup';
 
 export interface IStep {
   title: string;
   route: string;
   completed: boolean;
   canGoToStep: boolean;
+  validation?: {
+    schema: ObjectSchema<object>;
+    nameSpace: (currentTab: number) => string;
```

Comment:
We need this because the landform and the stepper both have 1/2 of the information needed to determine what the correct namespace is at runtime. So this callback functions allows the stepper to pass the tab back to the landform, which can then return the correct namespace.